plugins/outputs/stomp/stomp.go

```diff
@@ -0,0 +1,120 @@
+package stomp
+
+import (
+	ts "crypto/tls"
```

Comment:
I think you should not use the alias here
```suggestion
	"crypto/tls"
```